modules/nf-core/sourmash/gather/main.nf

```diff
@@ -0,0 +1,44 @@
+process SOURMASH_GATHER {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::sourmash=4.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/sourmash:4.5.0--hdfd78af_0':
+        'quay.io/biocontainers/sourmash:4.5.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(signature)
+    path(database)
+
+    output:
+    tuple val(meta), path('*.csv')           , optional:true, emit: result
```

Comment:
Is it correct this is optional? I would assume this would always contain some output? Or can it occur if there are no hits?

Reply:
I think that there could be cases without any hits (depends on a sample and reference db)